spec/extend-tests/subject-operator/200_test_extend_with_subject_transfers_subject_to_extender.hrx

```diff
@@ -2,6 +2,7 @@
 ---
 :ignore_for:
 - dart-sass
+- libsass
```

Comment:
Rather than marking these tests as ignored for all implementations, please just delete them.